R/utilities-parameter.R

```diff
@@ -120,11 +120,10 @@ setParameterValues <- function(parameters, values) {
 #' setParameterValuesByPath(c("Organism|Liver|Volume", "Organism|Volume"), c(2, 3), sim)
 #' @export
 setParameterValuesByPath <- function(parameterPaths, values, simulation) {
-  validateIsString(parameterPaths)
-  validateIsNumeric(values)
-  validateIsOfType(simulation, Simulation)
-  parameters <- sapply(parameterPaths, function(p) getParameter(p, simulation))
-  setParameterValues(parameters, values)
+  setQuantityValuesByPath(
+    quantityPaths = parameterPaths, values = values,

```

Comment:
I'd say either we put all parameters on the same line or one for each line. Here we are mixing and matching. Why?

Reply:
Hmm, I relied on styler on that..

Reply:
ok. if it's automated then I am fine